### ISSUE TITLE
Standard_Replication Tasks and Peeklock : Getting workflow saved successfully message along with validation error while trying to save When events are available for replication" trigger.

### ISSUE THREAD

**Issue by vgouth** [Azure/LogicAppsUX#2250]
### Describe the Bug with repro steps

**Prerequisites:**
Create a service bus namespace with two queues.
Create an event hub instance with two hubs.

1.Create a new Stateless logic app in standard, open the designer.
2.Add a event hub "When events are available for replication" trigger
3.In the trigger, enter the event hub name of your second event hub in "Event hub name"
4. Add the event hub "Replicate events" action. Enter the event hub name of your first event hub.
5.Save the workflow.
**Expected:** Workflow should save successfully.
**Actual:** Getting workflow saved successfully message along with validation error while trying to save When events are available for replication" trigger but workflow is not saving when we move to overview tab and come back to designer we can see workflow is not saved.

### What type of Logic App Is this happening in?

Standard (Portal)

### Are you using Preview Designer or GA Designer

Preview

### Workflow JSON

```json
{
  "definition": {
    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
    "actions": {
      "Replicate_events": {
        "type": "ServiceProvider",
        "inputs": {
          "parameters": {
            "eventHubName": "eh1",
            "skipAlreadyReplicated": false
          },
          "serviceProviderConfiguration": {
            "connectionName": "eventHub",
            "operationId": "replicateEvents",
            "serviceProviderId": "/serviceProviders/eventHub"
          }
        },
        "runAfter": {
          "builtin:newWorkflowTrigger": [
            "SUCCEEDED"
          ]
        }
      }
    },
    "contentVersion": "1.0.0.0",
    "outputs": {},
    "triggers": {
      "When_events_are_available_for_replication": {
        "type": "ServiceProvider",
        "inputs": {
          "parameters": {
            "eventHubName": "eh2"
          },
          "serviceProviderConfiguration": {
            "connectionName": "eventHub",
            "operationId": "receiveEventsForReplication",
            "serviceProviderId": "/serviceProviders/eventHub"
          }
        }
      }
    }
  },
  "connectionReferences": {
    "eventHub": {
      "connection": {
        "id": "/serviceProviders/eventHub/connections/eventHub"
      },
      "connectionName": "adw",
      "api": {
        "id": "/serviceProviders/eventHub"
      }
    }
  },
  "parameters": {}
}
```

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/113517384/236387040-60b5f0e0-17ec-4be6-8f29-76736c5bf71d.png)
![image](https://user-images.githubusercontent.com/113517384/236387771-07241c52-2453-4685-9e78-57b791ff420b.png)

### Browser

Edge

### Additional context

**Environment Details:**
**URL:** https://aka.ms/emastaging
**Version:** "2.30501.1.1"

**Comment by Eric-B-Wu** [comment]
@preetriti1 this is something we plan to implement where when workflow validation fails that we plan to prevent workflow saving, right?

**Comment by preetriti1** [comment]
We are not actually saving the workflow (so not a functional bug), just the message is wrong. I fixed the message but it will only be available in next release. We will still have to see why they are getting this validation error.

**Comment by preetriti1** [comment]
@vgouth - I am not able to reproduce this issue in staging for a newly created app with just adding trigger and action. Can you please try it again and reopen if this still reproduce?

**Comment by vgouth** [comment]
@preetriti1- Yes, I am not able to repro this issue in stating environment. But there is difference in version for stating portal ( 2.30410.1.12) and request URL portal (2.30501.1.1).  I am able to reproduce the above issue in request URL with current version (2.30501.1.5). 
![image](https://user-images.githubusercontent.com/113517384/237012896-0445fa25-d242-4fa3-a685-8aa9a2f7a628.png)

**NOTE:** @Eric-B-Wu -we don't have an option to reopen this issue, request you to please reopen it.

**Comment by Eric-B-Wu** [comment]
Staging Portal should be 2.30501.1.5, the request deployment had been updated with some fixes from 2.30501.1.1. If this is reproable in staging currently (post fixes) then I'll reopen it unless @preetriti1 this is fixed in 0508 release?

**Comment by vgouth** [comment]
Issue **not** repro in latest release.  Closing this issue

**Environment Details:**
**URL:** https://aka.ms/emastaging-flowstip1
**Version:** 2.30508.1.2
